packages/server/graphql/mutations/helpers/notifications/Notifier.ts

```diff
@@ -21,4 +21,10 @@ export type Notifier = {
     stageIndex: number,
     channelId: string
   ): Promise<NotifyResponse>
+  standupResponseSubmitted(
+    dataLoader: DataLoaderWorker,
+    meetingId: string,
+    teamId: string,
+    userId: string
+  ): Promise<void>
```

Comment:
+1 return val is `'success' | {error: Error}` i think?

Reply:
Nope! Most of the methods for the `Notifier` are treated as fire-and-forget, so we don't return anything here (`shareTopic` is different because the user action _is_ notifying, so it returns the result, but the other methods don't). It's the `NotifierIntegrationHelper` that returns something.

To be fair it's kind of a confusing pattern that I considered refactoring to be more readable as part of this, but I decided against investing the effort until we have a clearer sense of what upcoming integration work will look like.

Reply:
ohhh that's right. carry on!